parley/src/layout/editor.rs

```diff
@@ -555,106 +868,35 @@ where
         {
             self.generation.nudge();
         }
-        // Keeping this commented debug code in for now because it's quite
-        // useful when diagnosing selection problems:
-        //----------------------------------------------------------------------
-        // #[cfg(feature = "std")]
-        // {
-        //     let focus = new_sel.focus();
-        //     let cluster = focus.logical_clusters(&self.layout);
-        //     let dbg = (
-        //         cluster[0].as_ref().map(|c| &self.buffer[c.text_range()]),
-        //         focus.index(),
-        //         focus.affinity(),
-        //         cluster[1].as_ref().map(|c| &self.buffer[c.text_range()]),
-        //     );
-        //     print!("{dbg:?}");
-        //     let cluster = focus.visual_clusters(&self.layout);
-        //     let dbg = (
-        //         cluster[0].as_ref().map(|c| &self.buffer[c.text_range()]),
-        //         cluster[0]
-        //             .as_ref()
-        //             .map(|c| if c.is_word_boundary() { " W" } else { "" })
-        //             .unwrap_or_default(),
-        //         focus.index(),
-        //         focus.affinity(),
-        //         cluster[1].as_ref().map(|c| &self.buffer[c.text_range()]),
-        //         cluster[1]
-        //             .as_ref()
-        //             .map(|c| if c.is_word_boundary() { " W" } else { "" })
-        //             .unwrap_or_default(),
-        //     );
-        //     println!(" | visual: {dbg:?}");
-        // }
```

Comment:
This code seems to have disappeared. Is it no longer considered useful?

Reply:
Restored